code/parse/sexp.cpp

```diff
@@ -1762,6 +1762,16 @@ int check_operator_argument_count(int count, int op)
 	return 1;
 }
 
+template <typename T>
+int count_named_items(const char *name, int num, const T* item)
+{
+	int count = 0;
+	for (int i = 0; i < num; ++i)
+		if (!stricmp(name, item[i].name))
```

Comment:
Optional: `Assert(item[i].name != nullptr)`

Reply:
The name itself should have been caught before this point, but I added Asserts for the function arguments.